log-collector/src/main/scala/io/phdata/pulse/logcollector/Util.scala

```diff
@@ -0,0 +1,34 @@
+package io.phdata.pulse.logcollector
+
+import io.phdata.pulse.common.domain.LogEvent
+
+object Util {
+
+  /**
+   * Convert a [[LogEvent]] to a flattened [[Map[String,String]]
+   * @param logEvent LogEvent to convert
+   * @return The event, with properties flattened, as a [[Map[String,String]]
+   */
+  def logEventToFlattenedMap(logEvent: LogEvent): Map[String, String] = {
+    val values = logEvent.productIterator
```

Comment:
Since there are only 7 or 8 fields in a LogEvent, it might be easier and cleaner to just manually create a map from them.

This is a generic case class to Map function, but we only have a single case class, and we know all of the fields, so it seems a little complex for what we are trying to do

Reply:
Sounds good.  I wanted to avoid hardcoding information about the case class.  Will rewrite.

Reply:
Understand that. In this case hardcoding is fine. It will be obvious to a reader.